setup.py

```diff
@@ -17,55 +16,7 @@ def get_version():
 }
 extras["all"] = sum(extras.values(), [])
 
-install_requires = open("requirements.txt").read().strip().split("\n")
-
-setup(name = "uproot",
-      packages = find_packages(exclude = ["tests"]),
-      scripts = [],
-      version = get_version(),
-      author = "Jim Pivarski",
-      author_email = "pivarski@princeton.edu",
-      maintainer = "Jim Pivarski",
-      maintainer_email = "pivarski@princeton.edu",
-      description = "ROOT I/O in pure Python and NumPy.",
-      long_description = open("README.md").read(),
-      long_description_content_type = "text/markdown",
-      url = "https://github.com/scikit-hep/uproot4",
-      download_url = "https://github.com/scikit-hep/uproot4/releases",
-      license = "BSD 3-clause",
-      test_suite = "tests",
-      python_requires = ">=2.6, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*",
-      install_requires = ["numpy"],
-      tests_require = extras["test"],
```

Comment:
I've dropped these - `test_suite` and `tests_require` have been deprecated for a long time. `python setup.py test` requires `setup.py` and is an implementation detail, so using it to run tests was one of the first things to start getting the ax.

Reply:
Since we run `pytest` externally to any setup, this no longer has the semantic meaning that it once had. So sure, we can drop it.